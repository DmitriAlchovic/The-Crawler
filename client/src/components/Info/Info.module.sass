.info-container
    display: flex
    justify-content: space-between
    margin: 29px 107px 36px 133px

    .info
        width: 332px

        .header
            font-size: 18px
            font-weight: 600
            margin-bottom: 7px

        .body
            font-size: 16px
            color: #adadad
            word-wrap: wrap
    
