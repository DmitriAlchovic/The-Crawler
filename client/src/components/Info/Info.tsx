import React from 'react';
import styles from './Info.module.sass';

export default function Info() {
  return (
    <div className={styles['info-container']}>
      <div className={styles.info}>
        <p className={styles.header}>Best Prices & Deals</p>
        <p className={styles.body}>
          Don&apos;t miss our daily amazing deals prices
        </p>
      </div>
      <div className={styles.info}>
        <p className={styles.header}>Refundable</p>
        <p className={styles.body}>
          If your items have damage we agree to refund it
        </p>
      </div>
      <div className={styles.info}>
        <p className={styles.header}>Free delivery</p>
        <p className={styles.body}>
          Do purchase over $50 and get free delivery anywher
        </p>
      </div>
    </div>
  );
}
