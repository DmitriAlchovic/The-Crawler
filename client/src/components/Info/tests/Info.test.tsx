import React from 'react';
import { render } from '@testing-library/react';
import * as reduxHooks from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import Info from '../Info';

jest.mock('react-redux');

describe('Info', () => {
  it('should create empty component', () => {
    jest.spyOn(reduxHooks, 'useSelector').mockReturnValue('');
    const component = render(
      <BrowserRouter>
        <Info />
      </BrowserRouter>,
    );

    expect(component).toMatchSnapshot();
  });
});
