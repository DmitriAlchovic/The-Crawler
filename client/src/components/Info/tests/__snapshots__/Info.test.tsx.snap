// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info should create empty component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="info-container"
      >
        <div
          class="info"
        >
          <p
            class="header"
          >
            Best Prices & Deals
          </p>
          <p
            class="body"
          >
            Don't miss our daily amazing deals prices
          </p>
        </div>
        <div
          class="info"
        >
          <p
            class="header"
          >
            Refundable
          </p>
          <p
            class="body"
          >
            If your items have damage we agree to refund it
          </p>
        </div>
        <div
          class="info"
        >
          <p
            class="header"
          >
            Free delivery
          </p>
          <p
            class="body"
          >
            Do purchase over $50 and get free delivery anywhere
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="info-container"
    >
      <div
        class="info"
      >
        <p
          class="header"
        >
          Best Prices & Deals
        </p>
        <p
          class="body"
        >
          Don't miss our daily amazing deals prices
        </p>
      </div>
      <div
        class="info"
      >
        <p
          class="header"
        >
          Refundable
        </p>
        <p
          class="body"
        >
          If your items have damage we agree to refund it
        </p>
      </div>
      <div
        class="info"
      >
        <p
          class="header"
        >
          Free delivery
        </p>
        <p
          class="body"
        >
          Do purchase over $50 and get free delivery anywhere
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`ProfileMenu should create empty component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="info-container"
      >
        <div
          class="info"
        >
          <p
            class="header"
          >
            Best Prices & Deals
          </p>
          <p
            class="body"
          >
            Don't miss our daily amazing deals prices
          </p>
        </div>
        <div
          class="info"
        >
          <p
            class="header"
          >
            Refundable
          </p>
          <p
            class="body"
          >
            If your items have damage we agree to refund it
          </p>
        </div>
        <div
          class="info"
        >
          <p
            class="header"
          >
            Free delivery
          </p>
          <p
            class="body"
          >
            Do purchase over $50 and get free delivery anywhere
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="info-container"
    >
      <div
        class="info"
      >
        <p
          class="header"
        >
          Best Prices & Deals
        </p>
        <p
          class="body"
        >
          Don't miss our daily amazing deals prices
        </p>
      </div>
      <div
        class="info"
      >
        <p
          class="header"
        >
          Refundable
        </p>
        <p
          class="body"
        >
          If your items have damage we agree to refund it
        </p>
      </div>
      <div
        class="info"
      >
        <p
          class="header"
        >
          Free delivery
        </p>
        <p
          class="body"
        >
          Do purchase over $50 and get free delivery anywhere
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
